install-dev/autoload.php

```diff
@@ -23,17 +23,16 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
-
 spl_autoload_register(function ($className) {
     if (0 === strpos($className, 'InstallControllerConsole')) {
         $fileName = strtolower(str_replace('InstallControllerConsole', '', $className));
-        require_once __DIR__.'/controllers/console/' . $fileName . '.php';
+        require_once __DIR__ . '/controllers/console/' . $fileName . '.php';
```

Comment:
```suggestion
        require_once sprintf('%s/controllers/console/%s.php', __DIR__, $fileName);
```

it's very minor but I'd use sprintf for readability, since all those strings are edited by the PR anyway (but could be just me)

Reply:
Edited by cs fixer :sweat_smile:

Reply:
But I apply the fix